pkg/sidecar/sidecar.go

```diff
@@ -153,6 +155,22 @@ func (s *Sidecar) updateCertificates(svidResponse *workloadapi.X509Context) {
 // signalProcess sends the configured Renew signal to the process running the proxy
 // to reload itself so that the proxy uses the new SVID
 func (s *Sidecar) signalProcess() (err error) {
+	if s.config.PidFileName != "" {
+		atomic.StoreInt32(&s.processRunning, 1)
+		bytes, err := ioutil.ReadFile(s.config.PidFileName)
+		if err != nil {
+			return fmt.Errorf("Failed to read pid file: %s\n%w", s.config.PidFileName, err)
+		}
+		lines := strings.Split(string(bytes), "\n")
+		pid, err := strconv.Atoi(lines[0])
```

Comment:
why to parse only line 1? are you expecting to have more things in a pid file? if that is the case, is not that an error?

Reply:
is it not better to do:

```
	pid, err = strconv.Atoi(string(bytes.TrimSpace(bytes)))
```

Reply:
I'm not much of a go programmer. Took a rough stab at getting something working.

Your suggestion would work too I think and is simpler.